Formula/kubernetes-helm.rb

```diff
@@ -45,5 +48,6 @@ def install
 
     version_output = shell_output("#{bin}/helm version --client 2>&1")
     assert_match "GitTreeState:\"clean\"", version_output
+    assert_match stable.instance_variable_get(:@resource).instance_variable_get(:@specs)[:revision], version_output if build.stable?
```

Comment:
Yeah, I meant `if stable?`, sorry. The build part used to be effectively nil outside of the prepare/install stage. Maybe someone finally fixed that though.

_(Apologies for the repost; when I replied by email GitHub didn't stick the comment in the right discussion thread)_